### PR TITLE
Db w2

### PR DESCRIPTION
**Pull request by AdmWinther**

### CODE REVIEW

File: databases/week2/DB_W2_HW.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+INSERT INTO task 
+(title, description, created, updated, due_Date, status_id, user_id)
+VALUES 
+('Make a new task' ,
+'Make a new task in task table',
+NOW(),
+NOW(),
+NOW() + INTERVAL 5 DAY,
+1,
+1);
```
Comment (flemeur) [inline databases/week2/DB_W2_HW.sql:11]: Really cool that you are using INTERVAL combined with NOW()! :) 

My only suggestion would be to indent the column names and the values so they stand out visually. Something like this:

```sql
INSERT INTO task (
    title, description, created, updated, due_date, status_id, user_id
) VALUES (
    'Make a new task' ,
    'Make a new task in task table',
    NOW(),
    NOW(),
    NOW() + INTERVAL 5 DAY,
    1,
    1
);
```

File: databases/week2/DB_W2_HW.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+-- 2. Change the title of a task with these attributes: taskID, newTitle
+UPDATE task
+SET description = 'Change the title of a task'
+WHERE (id = 37);
+
+-- 3. Change the task due date with these attributes: taskID, newDueDate
+UPDATE task
+SET due_date = due_date + INTERVAL 20 DAY
+WHERE (id = 37);
+-- I intentionally were trying to see if I can add for example 20 days to the old due_date.
```
Comment (flemeur) [inline databases/week2/DB_W2_HW.sql:22]: Cool :+1: Again using INTERVAL can be useful

Comment (flemeur) [reply databases/week2/DB_W2_HW.sql:22]: Oh by the way: You don't need the parenthesis in the WHERE clause. It is usually only necessary in certain cases when combining AND and OR

File: databases/week2/DB_W2_HW2.sql [hunk 3]
```diff
@@ -0,0 +1,42 @@
+-- Create a new database containing the following tables
+CREATE DATABASE school;
+
+USE school;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+DROP TABLE class;
```
Comment (flemeur) [inline databases/week2/DB_W2_HW2.sql:7]: Just as a tip: You can do `DROP TABLE IF EXISTS class`. You could even do `DROP DATABASE IF EXISTS school` before you do `CREATE DATABASE school` that way you can run the entire sql file again and again and get a clean slate each time.

File: databases/week2/DB_W2_HW2.sql [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE TABLE student 
+(id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+name VARCHAR(50) NOT NULL,
+email VARCHAR(50),
+phone INTEGER(8),
+class_id INT UNSIGNED,
+PRIMARY KEY (id),
+FOREIGN KEY (class_id) REFERENCES class(id));
```
Comment (flemeur) [inline databases/week2/DB_W2_HW2.sql:23]: This is great. Again my only suggestion is indenting the column definitions for each table. Something like this:

```sql
CREATE TABLE student (
    id INT UNSIGNED NOT NULL AUTO_INCREMENT,
    name VARCHAR(50) NOT NULL,
    email VARCHAR(50),
    phone INTEGER(8),
    class_id INT UNSIGNED,
    PRIMARY KEY (id),
    FOREIGN KEY (class_id) REFERENCES class(id)
);
```

File: databases/week2/DB_W2_HW2.sql [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+( name , email , class_id)
+VALUES 
+('John Doe',
+'JD@example.dk',
+1);
+
+-- Add a new column to the class table named status which can only have 
+-- the following values: not-started, ongoing, finished (hint: enumerations).
+ALTER TABLE class
+ADD status ENUM ('not-started' , 'ongoing' , 'finished');
```
Comment (flemeur) [inline databases/week2/DB_W2_HW2.sql:42]: Nice. Remember that you could also add `NOT NULL DEFAULT 'not-started'`

**Review by flemeur — COMMENTED** [review]
Hi @hosseinTa 

Really great job on the homework! :1st_place_medal:

File: databases/week2/DB_W2_HW.sql (hunk 1 above)
Comment (AdmWinther) [reply databases/week2/DB_W2_HW.sql:11]: Hi Flemming,
Thanks a million for your advises. I will keep that in my mind.

File: databases/week2/DB_W2_HW.sql (hunk 2 above)
Comment (AdmWinther) [reply databases/week2/DB_W2_HW.sql:22]: You're right. It is just me being conservative and trying to see too far in the future ;)

File: databases/week2/DB_W2_HW2.sql (hunk 4 above)
Comment (AdmWinther) [reply databases/week2/DB_W2_HW2.sql:23]: Sure. Thanks

File: databases/week2/DB_W2_HW2.sql (hunk 5 above)
Comment (AdmWinther) [reply databases/week2/DB_W2_HW2.sql:42]: Great advise. Thanks. :)

**Pull request merged**